dist/module/hooks.mjs

```diff
@@ -129,7 +129,7 @@ export default function registerHooks() {
             const faceDown = deckCard.face === null;
             const whisper = game.settings.get('orcnog-card-viewer', 'enableWhisperCardTextToDM');
             const shareToAll = false;
-            if (id) new CardDealer('Deck of Many Things').view(id, faceDown, whisper, shareToAll);
+            if (id) new CardDealer(deckCard.source.name).view(id, faceDown, whisper, shareToAll);
```

Comment:
Already fixed this in https://github.com/orcnog/orcnog-card-viewer/pull/15/commits/d1068d1989775af43674bff7f8467f402fe5f4cd, but thank you